packages/libretro/sameboy/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="sameboy"
-PKG_VERSION="54f41cb"
+PKG_VERSION="44d8b16"
```

Comment:
We should address this issue with this [PR](https://github.com/libretro/Lakka-LibreELEC/pull/958) ? As it is not directly linked to adding support for Wandboard Quad (i.MX6).

Reply:
This is just fix for Lakka-2.3.2.

Reply:
I don't know, you say.

Reply:
Strange things happens in buildbot branch of libretro/SameBoy.
Few time ago i used libretro/SameBoy@c9e547c, but now it out of branch like libretro/SameBoy@54f41cb.
Look [this ](https://github.com/libretro/Lakka-LibreELEC/pull/926) please.

Reply:
It's normal, the repo is regularly rebased against upstream emulator and it breaks commits ids every time a rebase is done